.github/workflows/prcodecheck.yml

```diff
@@ -0,0 +1,27 @@
+name: Code style check
+
+on:
+  pull_request:
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  quality:
+    name: Code quality
+    uses: klepsydra-technologies/kpe-ghworkflows/.github/workflows/cppquality.yml@v2.0.6
```

Comment:
no cppquality for CE @tomasz3dk?

Reply:
Same as https://github.com/klepsydra-technologies/kpsr-sdk/pull/1#discussion_r1326079213

Reply:
Pretty sure the file has errors.